### PR TITLE
fix: Expand acceptance criteria to make them clear for 0011-NP-CLIE-client-interaction

### PR DESCRIPTION
**Pull request by MuthuVega**

### CODE REVIEW

File: non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md [hunk 1]
```diff
@@ -7,25 +7,28 @@ This specification contains a set of tests/acceptance criteria that clients (wal
...
+    - This is communicated to the wallet, and wallet uses the new parameter for each transaction tied to a block with a height higher than the one in which the change happened. The wallet should submit no more than `spam.pow.numberofTxPerBlock` number of transactions when tied to a block higher than the one in which the change happened. In case of excess transactions (i.e. greater than `spam.pow.numberofTxPerBlock`), transactions should not be sent and returns an appropriate error message OR delays sending the transactions. (<a name="0011-NP-CLIE-008" href="#0011-NP-CLIE-008">0011-NP-CLIE-008</a>)
```
Comment (Vegaklaus) [inline non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md]: I'd replace 'delays the transaction' with 'reprocesses the transaction (which may involve a new PoW and potentially delaying it)'. The reason is that it still may be possible to submit the transaction immediately if another past block is used for the spam protection, which would not be covered by the original formulation

File: non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md [hunk 2]
```diff
@@ -7,25 +7,28 @@ This specification contains a set of tests/acceptance criteria that clients (wal
...
+1. Create a key with sufficient tokens to be allowed to submit proposals and delegations (i.e., more than `SpamProtectionMinTokensForDelegation` and `SpamProtectionMinTokensForProposal`. Submit random (but valid) votes, delegation changes, transactions and proposals in sets of at least 13 each at the same time, at a rate of one set per second. Let that run for 4 epochs, with an epoch length of at least 100 blocks. Verify that the wallet returns an error for the excess transactions OR delays sending the transactions. Verify the account never gets banned, and that the client never crashed. (<a name="0011-NP-CLIE-018" href="#0011-NP-CLIE-018">0011-NP-CLIE-018</a>)
```
Comment (Vegaklaus) [inline non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md]: See above.

File: non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md [hunk 3]
```diff
@@ -7,25 +7,28 @@ This specification contains a set of tests/acceptance criteria that clients (wal
...
+1. Create 20 keys with sufficient tokens to be allowed proposals and delegations (i.e., more than `SpamProtectionMinTokensForDelegation` and `SpamProtectionMinTokensForProposal`. For all keys (within their respective wallets), submit random (but valid) votes, delegation changes, transactions and proposals in sets of at least 13 each at the same time, at a rate of one set per second. Let that run for 4 epochs, with an epoch length of at least 100 blocks. Verify that the wallet returns an error for the excess transactions OR delays sending the transactions. Verify that no account ever gets banned, and that the clients never crash. (<a name="0011-NP-CLIE-019" href="#0011-NP-CLIE-019">0011-NP-CLIE-019</a>)
```
Comment (Vegaklaus) [inline non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md]: See above

File: non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md (hunk 1 above)
Comment (MuthuVega) [reply non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md]: Amendments made as suggested above.

File: non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md (hunk 2 above)
Comment (MuthuVega) [reply non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md]: Amendments made as suggested above.

File: non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md (hunk 3 above)
Comment (MuthuVega) [reply non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md]: Amendments made as suggested above.

**Review by vega-paul — APPROVED** [review]

**Pull request merged**